browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -354,6 +354,18 @@ test.describe('End to end enumerator test', () => {
           page.locator('#enumerator-field-add-button'),
         ).toHaveAttribute('disabled')
       })
+
+      await test.step('Add button is still disabled after trying to save', async () => {
+        await applicantQuestions.clickNext()
+        // Verify that the error message is visible.
+        await expect(
+          page.locator('.cf-applicant-question-errors:visible'),
+        ).toContainText('Error: Please enter a value for each line.')
```

Comment:
suggestion (non-blocking):  As much as possible, it's best to avoid relying on text either for locators or assertions because if that text changes, we'll have to update the test.  You could use `.toBeVisible()` instead.

Reply:
Interesting point - the unit tests already validate the error message text and that the errors occur at the right times, so the browser tests are somewhat duplicative in that sense (I copied this from a test around line 200). So I guess the question is how much should the browser tests test - a screenshot with a single error case (or two at once, maybe) could validate that (and how) error messages are presented in general.

But actually in this case I added this more so that the test reads clearly and to confirm that the page is in the expected state when validating that the add button is disabled (i.e. we're not actually on the next page where it is disabled for other reasons). I guess toBeVisible with a comment is sufficient to cover that. Updated to do that.

Thanks!